NickvisionMoney.Shared/Docs/yelp/es/import-export.page

```diff
@@ -23,15 +23,75 @@
 	<list>
 		<item>
 			<p>Open Financial Exchange (<file>.ofx</file>)</p>
+			<p>Las fechas deben estar en formato <em>AAAMMDD</em>.</p>
 		</item>
 		<item>
 			<p>Quicken Interchange Format (<file>.qif</file>)</p>
+			<p>Las fechas deben estar en formato inglés (EE. UU.) (<em>MM/DD/AAAA</em>).</p>
 		</item>
 		<item>
 			<p>Denaro CSV (<file>.csv</file>)</p>
 			<p>❗ Si bien CSV es un formato común y algunos bancos permiten a sus usuarios exportar información como CSV, estos archivos son, de hecho, tablas que pueden contener datos arbitrarios. La importación de un archivo CSV creado en otro programa fallará porque sus datos no serán compatibles con lo que <app>Denaro</app> intenta obtener de un archivo. El propósito de importar/exportar CSV es proporcionar una forma de agregar datos a una cuenta mediante programas externos, como editores de texto y paquetes de oficina. La importación solo agregará nuevas transacciones sin anular las existentes. Si desea crear un archivo CSV manualmente, este es el encabezado que incluye todas las columnas que <app>Denaro</app> espera encontrar:</p>
 			<p><code>ID;Fecha;Descripción;Tipo;IntervaloRepetición;RepetirDesde;RepetirFechaFin;Cantidad;RGBA;Grupo;NombreGrupo;DescripciónGrupo</code></p>
```

Comment:
@nlogozzo and me discussed it and deicided that it would be better if column titles are kept untranslated, since in exported CSV files they're always in English.

Reply:
And then when each column is described you can do something like `Date = Fecha`, `Description = Descripción`, etc...

Reply:
And then when each column is described you can do something like `Date = Fecha`, `Description = Descripción`, etc...

Reply:
And then when each column is described you can do something like `Date = Fecha`, `Description = Descripción`, etc...

Reply:
I'm not sure about `Date = Fecha` etc, I think it's not a problem to keep item titles in English since there are descriptions (and these descriptions can be modified if needed to not be translated literally but contain good explanation)

Reply:
It should already be changed, let me know if any other changes are needed 😄